doc/input-output-reference/src/overview/group-node-branch-management.tex

```diff
@@ -443,6 +443,125 @@ \subsubsection{Outputs}\label{outputs-3-012}
 
 No output directly related to Connector Lists is currently presented.
 
+\subsection{Plant-Condenser Flow Control}\label{plant-condenser-flow-control}
```

Comment:
This is the right place for the Connector:Splitter/Mixer docs, but the tempering valve doesn't belong here. The "Plant Condenser Flow Control" main heading and the tempering valve should be after the Pumps (as in the IDD groupings). This heading here can be changed to, say, "Plant Loop Connectors"?

Reply:
Did as recommended.  Moved the Plant Condenser Flow Control to its own section (new file) right after the pump section.  Added an intro sentence for the new section.  Changed the name of what stayed where it got up to Plant Loop Connectors.  Will show up in the next commit.